devnet-sdk/contracts/constants/constants.go

```diff
@@ -6,10 +6,43 @@ import (
-	SuperchainTokenBridge      types.Address = common.HexToAddress("0x4200000000000000000000000000000000000028")
+	L2ToL1MessagePasser           types.Address = common.HexToAddress("0x4200000000000000000000000000000000000016")
+	DeployerWhitelist             types.Address = common.HexToAddress("0x4200000000000000000000000000000000000002")
+	WETH                          types.Address = common.HexToAddress("0x4200000000000000000000000000000000000006")
+	L2CrossDomainMessenger        types.Address = common.HexToAddress("0x4200000000000000000000000000000000000007")
+	L2StandardBridge              types.Address = common.HexToAddress("0x4200000000000000000000000000000000000010")
+	SequencerFeeVault             types.Address = common.HexToAddress("0x4200000000000000000000000000000000000011")
+	OptimismMintableERC20Factory  types.Address = common.HexToAddress("0x4200000000000000000000000000000000000012")
+	L1BlockNumber                 types.Address = common.HexToAddress("0x4200000000000000000000000000000000000013")
+	GasPriceOracle                types.Address = common.HexToAddress("0x420000000000000000000000000000000000000F")
+	L1Block                       types.Address = common.HexToAddress("0x4200000000000000000000000000000000000015")
+	GovernanceToken               types.Address = common.HexToAddress("0x4200000000000000000000000000000000000042")
+	LegacyMessagePasser           types.Address = common.HexToAddress("0x4200000000000000000000000000000000000000")
+	L2ERC721Bridge                types.Address = common.HexToAddress("0x4200000000000000000000000000000000000014")
+	OptimismMintableERC721Factory types.Address = common.HexToAddress("0x4200000000000000000000000000000000000017")
+	ProxyAdmin                    types.Address = common.HexToAddress("0x4200000000000000000000000000000000000018")
+	BaseFeeVault                  types.Address = common.HexToAddress("0x4200000000000000000000000000000000000019")
+	L1FeeVault                    types.Address = common.HexToAddress("0x420000000000000000000000000000000000001a")
+	SchemaRegistry                types.Address = common.HexToAddress("0x4200000000000000000000000000000000000020")
+	EAS                           types.Address = common.HexToAddress("0x4200000000000000000000000000000000000021")
+	CrossL2Inbox                  types.Address = common.HexToAddress("0x4200000000000000000000000000000000000022")
+	L2ToL2CrossDomainMessenger    types.Address = common.HexToAddress("0x4200000000000000000000000000000000000023")
+	SuperchainWETH                types.Address = common.HexToAddress("0x4200000000000000000000000000000000000024")
+	ETHLiquidity                  types.Address = common.HexToAddress("0x4200000000000000000000000000000000000025")
+	SuperchainTokenBridge         types.Address = common.HexToAddress("0x4200000000000000000000000000000000000028")
+	Create2Deployer               types.Address = common.HexToAddress("0x13b0D85CcB8bf860b6b79AF3029fCA081AE9beF2")
```

Comment:
just for confirmation, those are stable addresses too ?

Reply:
yes - they are defined as constants here devnet-sdk/contracts/constants/constants.go for go and here packages/contracts-bedrock/src/libraries/Preinstalls.sol for Solidity